Akka.Serialization.MessagePack/MsgPackSerializer.cs

```diff
@@ -173,7 +177,21 @@ public override object FromBinary(byte[] bytes, Type type)
             return MessagePackSerializer.Deserialize(type, bytes,SerializerOptions);
         }
 
-        public override int Identifier => 151;
+        public override int Identifier
+        {
+            get
+            {
+                if (_serializerId != -1)
+                    return _serializerId;
+                
+                var config = system.Settings.Config.GetConfig(SerializationIdentifiers);
+                var identifiers = config.AsEnumerable()
+                    .ToDictionary(pair => Type.GetType(pair.Key, true), pair => pair.Value.GetInt());
+                _serializerId = identifiers.TryGetValue(GetType(), out var value) ? value : DefaultSerializerId;
+                
+                return _serializerId;
+            }
+        }
```

Comment:
Do not hard code the serializer id, detect if it is being overriden inside the HOCON config.

Reply:
The one hesitation I had with a setup like this,
Does it allow someone to read old records from old ID while also ensuring new records are written with new ID?

Reply:
No, everything will be read and written using the overriden ID